admin/app/adapters/scoring-and-capacity-simulator-report.js

```diff
@@ -0,0 +1,18 @@
+import ApplicationAdapter from './application';
+
+export default class ScoringAndCapacitySimulatorReportAdapter extends ApplicationAdapter {
+  namespace = 'api/admin';
+
+  getSimulationResult({ score, capacity }) {
+    let data;
+    if (score) {
+      data = { score };
+    } else if (capacity) {
+      data = { capacity };
+    }
+    const url = `${this.host}/${this.namespace}/simulate-score-or-capacity`;
+    return this.ajax(url, 'POST', {
+      data: { data },
```

Comment:
On peut pas éviter le if/else en faisant ceci directement ?
```suggestion
      data: { score, capacity },
```

Reply:
On a implémenter ceci à cause du xor, sur la validation joy de la route. La route s'attend soit à un objet avec `capacity` en clé soit `score`, mais pas les deux. Si on envoie un objet `{ score: 200, capacity: undefined }` ou l'inverse, on se prend une 400